packages/calypso-e2e/src/lib/pages/cart-checkout-page.ts

```diff
@@ -75,6 +79,21 @@ export class CartCheckoutPage {
 		this.page = page;
 	}
 
+	/**
+	 * Navigates to checkout page of the specified blog.
+	 *
+	 * @param {{path: string}: string } param1 Key/value pair of the blogname for which checkout is to be loaded.
+	 */
+	async visit( { blogName }: { blogName: string } = { blogName: '' } ): Promise< void > {
```

Comment:
If `blogName` is optional, this would result in the following screen being shown unexpectedly:

![image](https://user-images.githubusercontent.com/6549265/149017754-f9644ab1-e3e9-4875-adb7-903487a62ff7.png)

I think a more robust solution would be to require the parameter `blogName`. What do you think?

Reply:
Makes sense, I've refactored as you suggested.